bridge/framework/django.py

```diff
@@ -33,17 +34,28 @@ def configure_postgres(self, platform: Platform) -> None:
             }
         }
 
+    def configure_redis(self, config: RedisConfig) -> None:
+        self.framework_locals["CACHES"] = {
```

Comment:
Just configuring it as a cache to validate it works, we can roll this back to only celery or make this configurable by the end user if we want